core/src/sumeragi/message.rs

```diff
@@ -78,20 +77,14 @@ impl From<&ValidBlock> for BlockSigned {
 #[derive(Debug, Clone, Decode, Encode)]
 #[non_exhaustive]
 pub struct BlockCommitted {
-    /// Hash of the block being signed.
-    pub hash: HashOf<SignedBlock>,
```

Comment:
I think that hash of block is still helpful for diagnostic, we can reject message not because signature is wrong, but because it was sent for different block entirely.

Same for `BlockSigned` message.

Reply:
it's not quite the same for `BlockSigned` since those can come before `BlockCreated`. I can agree about the `BlockCommitted` though

Reply:
done